openwisp_monitoring/monitoring/migrations/__init__.py

```diff
@@ -37,6 +37,38 @@ def assign_permissions_to_groups(apps, schema_editor):
             )
 
 
+def assign_inline_permissions_to_groups(apps, schema_editor):
+    create_default_permissions(apps, schema_editor)
+    operators_read_only_admins_manage = [
+        'check',
+        'alertsettings',
+    ]
+    manage_operations = ['add', 'change', 'delete']
+    Group = get_swapped_model(apps, 'openwisp_users', 'Group')
+
+    try:
+        admin = Group.objects.get(name='Administrator')
+        operator = Group.objects.get(name='Operator')
+    # consider failures custom cases
+    # that do not have to be dealt with
+    except Group.DoesNotExist:
+        return
+
+    for model_name in operators_read_only_admins_manage:
+        try:
+            permission = Permission.objects.get(
+                codename='view_{}_inline'.format(model_name)
+            )
+            operator.permissions.add(permission.pk)
```

Comment:
We should also add `view` permission to the administrator for completeness.

Reply:
Sure :+1: